lib/logstash/outputs/dynatrace.rb

```diff
@@ -17,6 +17,7 @@
 require 'logstash/namespace'
 require 'logstash/outputs/base'
 require 'logstash/json'
+require 'openssl'
 
 MAX_RETRIES = 5
 PLUGIN_VERSION = '0.3.2'
```

Comment:
```suggestion
PLUGIN_VERSION = '0.4.0'
```

Reply:
Minor? This isn't adding a new feature but fixing a bug. I'd go with patch

Reply:
The explicit single concurrency and adapted error handling might still change behavior for someone relying on it.